src/WebJobs.Extensions.DurableTask.Scale/AzureStorage/AzureStorageScalabilityProviderFactory.cs

```diff
@@ -0,0 +1,223 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Azure.WebJobs.Host.Scale;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Scale.AzureStorage
+{
+    /// <summary>
+    /// Factory class responsible for creating <see cref="AzureStorageScalabilityProvider"/> instances.
+    /// </summary>
+    public class AzureStorageScalabilityProviderFactory : IScalabilityProviderFactory
+    {
+        private const string LoggerName = "Host.Triggers.DurableTask.AzureStorage";
```

Comment:
Why do we use the Host.Triggers prefix? Is this a scale controller convention? If so, we should add a comment explaining this because the we're not actually running in a "Host" context, so this choice of category name could be misleading.

Reply:
removed, sry I forgot to delete it..